packages/web-pkg/src/components/FilesList/ResourceTable.vue

```diff
@@ -198,11 +198,21 @@
         @click="openSharingSidebar(item)"
       >
         <oc-avatars
-          class="resource-table-people"
-          :items="getSharedByAvatarItems(item)"
+          class="resource-table-people oc-flex"
           :is-tooltip-displayed="true"
+          :items="getSharedByAvatarItems(item)"
           :accessible-description="getSharedByAvatarDescription(item)"
-        />
+        >
+          <template #userAvatars>
+            <user-avatar
+              v-for="avatar in getSharedByAvatarItems(item)"
+              :key="avatar.username"
+              :user-id="avatar.id"
+              :user-name="avatar.displayName"
+              :width="30"
+            />
```

Comment:
@kulmann  
I can't think of a smarter solution here, that won't be blocking (await)

Reply:
Thinking more and more about it, it's rather smart to do it that way 🤓